src/neuralnet/tally.cpp

```diff
@@ -490,7 +491,7 @@ class ResearcherTally
         // starting height yet, so the tally will initialize without it.
         //
         if (m_start_pindex == nullptr && pindexGenesisBlock != nullptr) {
-            const int32_t threshold = GetResearchAgeThreshold();
+            const int32_t threshold = Params().GetConsensus().BlockV11Height;
```

Comment:
I think this should be ResearchAgeHeight, right?

Reply:
Yep, will fix.